tmt/steps/execute/internal.py

```diff
@@ -151,11 +153,31 @@ def execute(self, test: Test, guest: Guest,
         # Create data directory, prepare test environment
         environment = self._test_environment(test, extra_environment)
 
+        test_wrapper_filepath = os.path.join(workdir, TEST_WRAPPER_FILENAME)
+
         # Prepare the test command (use default options for shell tests)
         if test.framework == "shell":
-            command = f"{tmt.utils.SHELL_OPTIONS}; {test.test}"
+            test_command = f"{tmt.utils.SHELL_OPTIONS}; {test.test}"
+        else:
+            test_command = test.test
+
+        self.write(test_wrapper_filepath, test_command, 'w')
+
+        os.chmod(test_wrapper_filepath, 0o755)
+
+        guest.push(
+            source=test_wrapper_filepath,
+            destination=test_wrapper_filepath,
+            options=["-p", "--chmod=755"]
+            )
+
+        remote_command = f'./{TEST_WRAPPER_FILENAME}'
+
+        if self.get('interactive'):
+            pass
```

Comment:
What's the point of this `pass` here?

Reply:
No longer present in 45e9bd5c654d8c31645fc2e630f2d4ebcfdbd28d.